src/shared/contexts/editor/index.tsx

```diff
@@ -66,7 +66,9 @@ export const EditorProvider: FC = ({children}) => {
   const injectViaLsp = useCallback(
     (cmd, data: Omit<ExecuteCommandArgument, 'textDocument'>) => {
       try {
-        connection.current.inject(cmd, data)
+        if (connection.current && !isFlagEnabled('schemaComposition')) {
+          connection.current.inject(cmd, data)
+        }
```

Comment:
We do not need to featureFlag the existing use of `injectViaLsp()` in the schema browser. Since it's all feature flagged in this single place.